packages/web-client/app/components/card-pay/deposit-workflow/confirmation/index.hbs

```diff
@@ -39,8 +39,8 @@
   <div class="card-pay-confirmation__bridge-container">
     <div class="card-pay-confirmation__bridge-inner">
       Bridge
-      <Boxel::Button @as="anchor" @size="extra-small" href={{this.bridgeExplorerUrl}} class="card-pay-confirmation__bridge-inner-button">
-        View in Bridge Explorer
+      <Boxel::Button @as="anchor" @size="extra-small" href={{this.bridgeExplorerUrl}} target="_blank" rel="noopener" class="card-pay-confirmation__bridge-inner-button">
+        View AMB receipt
```

Comment:
This is actually changed to "View in Bridge Explorer" in the script. Both of these lead to the same link, and no one seems to know why we use different text.

Reply:
Good catch!